### PR TITLE
Fix #10173 - Prevent lag in external monitors when using Mac + Chrome

### LINKED ISSUE #10173: Mitigate CSS transition lag Chromium bug on macOS external monitors
There is a [Chromium bug](https://bugs.chromium.org/p/chromium/issues/detail?id=971701) that causes extreme lag in CSS transitions on external monitors on macOS. There don't seem to be any great workarounds, other than manually forcing the page to re-draw on an interval or avoiding CSS transitions altogether.

This issue is meant to consolidate efforts to mitigate or workaround this issue, since it has been brought up many times as different symptoms have been reported. I'll mark it as "blocked" for now, unless someone comes up with a viable mitigation at least.

Related issues:
* #8324
* #10202

### PR DESCRIPTION
**Pull request by darkwing**
Fixes: #10173

Explanation:  Add a bit of CSS that renders a visually invisible rotating 1px element that forces the UI to update on each tick

Manual testing steps:  
  - Move Chrome window to an external monitor
  - Open Metamask popup and type in any input (password or a send amount)
  - See the UI updating after each keystroke

### CODE REVIEW

**Review by Gudahtt — COMMENTED** [review]
Code looks good to me! Hold off on approving so that someone with macOS can test this to verify it works.

**Review by brad-decker — APPROVED** [review]
works for me and my old setup, no input lag where its clearly present in prod. Just a comment about edge support but not sure if it matters?

File: app/scripts/metamask-controller.js
Comment (darkwing) [reply app/scripts/metamask-controller.js:486]: Since both Edge and Chrome are Chromium-based, I would expect this cure Edge's woes as well!

Comment (brad-decker) [reply app/scripts/metamask-controller.js:486]: sounds good to me

**Pull request merged**
